RLBotDiscordBot/bot.py

```diff
@@ -46,15 +48,19 @@ def __init__(self):
     async def on_ready(self):
         self.logger.info(f'{self.user} online! (ID: {self.user.id})')
         self.has_reacted = 0
-        self.has_checked = False
 
     async def on_message(self, message):
+
         if not self.has_checked:
```

Comment:
This should still be in on_ready or somewhere that is called only once when the bot is started. No reason for it to be in on_message.

Also consider scheduling this so it runs every week or so.